code/modules/antagonists/cult/cult.dm

```diff
@@ -131,7 +131,9 @@
 		H.eye_color = initial(H.eye_color)
 		H.dna.update_ui_block(DNA_EYE_COLOR_BLOCK)
 		H.remove_trait(CULT_EYES)
-		H.update_body()
+	current.cut_overlays()
+	current.regenerate_icons()
```

Comment:
i cri eritiem

Reply:
The blood halo appears for Xeno-Cultists and other wacky non-humans too, update_body doesn't work and even if it did it wouldn't cover all the possible cases.